documentation/how-to/variables.en.md

```diff
@@ -1,23 +1,50 @@
 ---
-title: Global variables
+title: Variables
 tx_slug: documentation_how-to_variables
 ---
 
-# Global variables
+# Variables
 
-Variables are used in QField and QGIS to dynamically set values based on
-the current project file, the layer or the system. They can be used for
-styling, setting default values for fields and much more.
+QField allows user to add and edit expression variables.
+These can be used for a wide range of applications, including data-defined symbology, setting default field values, controlling print layout and application behavior, and much more.
+See [the relevant QGIS documentation](https://docs.qgis.org/3.40/en/docs/user_manual/introduction/general_tools.html#storing-values-in-variables) for more information.
```

Comment:
Tip: instead of using a fixed QGIS version (i.e. in this case /3.40/), we should use /latest/ to insure our documentation ages alongside QGIS :)

```suggestion
See [the relevant QGIS documentation](https://docs.qgis.org/latest/en/docs/user_manual/introduction/general_tools.html#storing-values-in-variables) for more information.
```